source/reference/methods.txt

```diff
@@ -762,6 +774,53 @@ Server Status Methods
 
      - Returns the version of the :binary:`~mongod` instance.
 
+Sharding Methods
+----------------
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+
+     - Description
+
+   * - :method:`sh.addShard()`
+
+     - Adds a shard to a sharded cluster.
+
+   * - :method:`sh.addShardTag()`
+
+     - Associates a shard with a tag or identifier. In MongoDB 3.4 and
```

Comment:
As I commented on the other thread, you might want to leave this out because it just aliases to addShardToZone since mongosh doesn't officially support v3.4 and before.

Reply:
Done.